bucket/gnupg.json

```diff
@@ -1,15 +1,30 @@
 {
     "homepage": "https://www.gnupg.org/",
+    "description": "An open source implementation of PGP standard, can encrypt and sign data and communications.",
     "version": "2.2.17",
     "license": "GPL-3.0-or-later",
     "url": "https://www.gnupg.org/ftp/gcrypt/binary/gnupg-w32-2.2.17_20190709.exe#/dl.7z",
-    "hash": "e5c99867591c1f72190f5540040ba027233f8cf5acc9ed627dba89a2461b57e0",
+    "hash": "sha1:82abfbc79d1a99b27f25ba92fe878cad07a31532",
+    "pre_install": [
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\gnupg-uninstall.exe.nsis\" -Force -Recurse",
+        "New-Item \"$dir\\bin\\gpgconf.ctl\" -Force | Out-Null",
+        "if (!(Test-Path \"$persist_dir\\home\") -and (Test-Path \"$Env:Appdata\\gnupg\")) {",
+        "    Write-Host -F yellow \"Copying old '$Env:Appdata\\gnupg' to '$persist_dir\\home'\"",
```

Comment:
Keep consistency between enum quoting. Either quote it everywhere or do not quote it anywhere.  L13 Directory